File: PastaPHP/Model/Aluno.php
<?php
    class Aluno extends Professor {  #extende usuario + rgm
        private $curso;
        private $anoLetivo;

        public function __construct($nome, $id, $cpf, $email, $senha, $rgm, $curso, $anoLetivo){
            parent::__construct($nome, $id, $cpf, $email, $senha, $rgm);
            $this->curso = $curso;
            $this->anoLetivo = $anoLetivo;
        }     

     public function getCurso(){
        return $this->curso;
    }
    
    public function setCurso($curso){
        $this->curso = $curso; 
    }

     public function getAnoLetivo(){
        return $this->anoLetivo;
    }
    
    public function setAnoLetivo($anoLetivo){
        $this->anoLetivo = $anoLetivo; 
    }

   
      public function __toString(){
        return parent::__toString(). "Curso: {$this->curso} - Ano Letivo: {$this->anoLetivo}";
    }

    }
?>

File: PastaJava/Model/Aluno.java
public class Aluno extends UsuarioSistema {
    private String curso;
    private String anoLetivo;

    public Aluno() {}

    public Aluno(String nome, int id, String cpf, String email, String senha, String curso, String anoLetivo) {
        super(nome, id, cpf, email, senha);
        this.curso = curso;
        this.anoLetivo = anoLetivo;
    }

    public String getCurso() {
        return curso;
    }

    public void setCurso(String curso) {
        this.curso = curso;
    }

    public String getAnoLetivo() {
        return anoLetivo;
    }

    public void setAnoLetivo(String anoLetivo) {
        this.anoLetivo = anoLetivo;
    }

    @Override
    public String toString() {
        return super.toString() + " Aluno [curso=" + curso + ", anoLetivo=" + anoLetivo + "]";
    }
}
